lib/spack/spack/build_systems/sip.py

```diff
@@ -67,7 +67,7 @@ def import_modules(self):
         modules = []
         root = os.path.join(
             self.prefix,
-            self.spec['python'].package.config_vars['python_lib']['true']['false'],
+            self.spec['python'].package.purelib,
```

Comment:
This is actually a behavior change (platlib -> purelib). I'm not sure which is better or if both need to be used. This only affects RHEL/CentOS/Fedora system Python and sip packages, of which there are very few.

Reply:
Took a look through the source code and it looks like platlib is the correct answer:
```python
        Option('target_dir', default=sysconfig.get_path('platlib'),                      
                help="the target installation directory", metavar="DIR",                 
                tools=['build', 'install']),                     
```